### DISCUSSION TITLE
Add custom colors for scene buttons?

### DISCUSSION THREAD

**Discussion by kkroc** [Kounex/obs_station#59] [Ideas]
Is there a way for you to add selectable colors for Scene buttons?   or would this be a OBS feature?  

It would great to assign a color to certain Scene buttons to help differentiate them when there are many of them on the screen at once. 

Thanks!

**Comment by Kounex** [comment]
That's definitely doable! Do you know if something like that is possible in OBS itself out of the box?

**Comment by kkroc** [comment]
It isn’t a function in OBS and I’ve looked and don’t see a plug-in either.  Thanks!On Jul 9, 2023, at 4:18 PM, René Schramowski ***@***.***> wrote:﻿
That's definitely doable! Do you know if something like that is possible in OBS itself out of the box?
